lisa/util/constants.py

```diff
@@ -134,3 +134,5 @@
 DEVICE_TYPE_SRIOV = "SRIOV"
 DEVICE_TYPE_NVME = "NVME"
 DEVICE_TYPE_GPU = "GPU"
+
+FIO_MODES = "randread randwrite read write"
```

Comment:
1. Don't put tool specified constants in the global constants file.
2. Define it as Enum.

Reply:
updated